jina/flow/base.py

```diff
@@ -1753,6 +1667,47 @@ def rolling_update(
             any_event_loop=True,
         )
 
+    def to_k8s_yaml(
+        self,
+        output_base_path: str,
+        k8s_namespace: Optional[str] = None,
+        k8s_connection_pool: bool = True,
```

Comment:
The prefix `k8s_`  of parameter names seems not necessary anymore, since `to_k8s_yaml` only works for k8s.